examples/data-objects/codemirror/src/index.ts

```diff
@@ -72,7 +72,9 @@ class CodeMirrorFactory extends RuntimeFactoryHelper {
             buildRuntimeRequestHandler(
                 mountableViewRequestHandler(MountableView, [viewRequestHandler]),
             ),
-            undefined, // runtimeOptions
+            {
+                summaryOptions: { summaryConfigOverrides: { state: "disabled" } },
```

Comment:
Should this be removed, given you mentioned summaries work now?

Reply:
Didn't mean to commit this as it is only relevant to testing
I haven't done example testing with summaries yet, but the main areas should be covered